### PR TITLE
[WEB-1992] Design updates

### PR DESCRIPTION
**Pull request by dpiatek**
Jira Ticket Link / Motivation
----

https://ably.atlassian.net/browse/WEB-1992

Summary of changes
----

See commits for details. 

How do you manually test this?
----

All changes can be seen on the review app.

Reviewer Tasks (optional)
----

Merge/Deploy Checklist
----

- [ ] Written automated tests for implemented features/fixed bugs
- [ ] Rebased and squashed commits
- [ ] Commits have clear descriptions of their changes
- [ ] Checked for any performance regressions

Frontend Checklist
----

- [ ] No frontend changes in this PR
- [ ] Added before/after screenshots for changes
- [ ] Tested on different platforms/browsers with [Browserstack](https://www.browserstack.com)
- [ ] Compared with the initial design / our [brand guidelines](https://www.figma.com/file/jTQgyIovrhGBOf4Zrvjvbk/Ably-Linear-Design?node-id=118%3A0)
- [ ] Checked the code for accessibility issues ([VoiceOver User Guide](https://support.apple.com/en-gb/guide/voiceover/welcome/mac))?

### CODE REVIEW

**Review by fliptopbox — APPROVED** [review]
LGTM, some minor comments but nothing worth blocking the approval.

File: src/core/icons/icon-display-servers.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
...091 25 48 26.7909 48 29V37C48 39.2091 46.2091 41 44 41H4C1.79086 41 0 39.2091 0 37V29ZM6.75 30C6.75 29.5858 6.41421 29.25 6 29.25C5.58579 29.25 5.25 29.5858 5.25 30V36C5.25 36.4142 5.58579 36.75 6 36.75C6.41421 36.75 6.75 36.4142 6.75 36V30ZM9 29.25C9.41421 29.25 9.75 29.5858 9.75 30V36C9.75 36.4142 9.41421 36.75 9 36.75C8.58579 36.75 8.25 36.4142 8.25 36V30C8.25 29.5858 8.58579 29.25 9 29.25ZM41 34C41.5523 34 42 33.5523 42 33C42 32.4477 41.5523 32 41 32C40.4477 32 40 32.4477 40 33C40 33.5523 40.4477 34 41 34ZM36 33C36 33.5523 35.5523 34 35 34C34.4477 34 34 33.5523 34 33C34 32.4477 34.4477 32 35 32C35.5523 32 36 32.4477 36 33ZM6 11.25C6.41421 11.25 6.75 11.5858 6.75 12V18C6.75 18.4142 6.41421 18.75 6 18.75C5.58579 18.75 5.25 18.4142 5.25 18V12C5.25 11.5858 5.58579 11.25 6 11.25ZM9.75 12C9.75 11.5858 9.41421 11.25 9 11.25C8.58579 11.25 8.25 11.5858 8.25 12V18C8.25 18.4142 8.58579 18.75 9 18.75C9.41421 18.75 9.75 18.4142 9.75 18V12ZM42 15C42 15.5523 41.5523 16 41 16C40.4477 16 40 15.5523 40 15C40 14.4477 40.4477 14 41 14C41.5523 14 42 14.4477 42 15ZM35 16C35.5523 16 36 15.5523 36 15C36 14.4477 35.5523 14 35 14C34.4477 14 34 14.4477 34 15C34 15.5523 34.4477 16 35 16Z" fill="#03020D"/>
```
[the remark this answers is not in the document]
Comment (dpiatek) [reply src/core/icons/icon-display-servers.svg:2]: yup that's correct, only the gui icons need `currentColor`

File: src/core/styles/properties.css [hunk 2]
```diff
@@ -45,10 +45,27 @@
...
+      #f1280a 78.13%,
+      #e90f04 88.02%,
+      var(--color-red-orange) 92.73%
+    );
+
+    /* Used for smooth transitions from gradient to non-gradient backgrounds */
+    --gradient-transparent: linear-gradient(
+      0deg,
+      rgba(255, 255, 255, 0),
+      rgba(255, 255, 255, 0)
```
[the remark this answers is not in the document]
Comment (dpiatek) [reply src/core/styles/properties.css:62]: Did try this, looks like at least two colour stops are needed!

[the remark this answers is not in the document]
Comment (fliptopbox) [reply src/core/styles/properties.css:62]: oh. TIL

**Pull request merged**